packages/code-editor/src/CodeEditor/languages/xml.ts

```diff
@@ -386,3 +388,13 @@ export const hvXmlFormatter = async (
         ...options,
       });
 };
+
+export interface XmlLanguagePlugin extends Omit<LanguagePlugin, "schema"> {}
+
+export const xmlLanguagePlugin: XmlLanguagePlugin = {
```

Comment:
we don't have `"schema"` anymore, the plugin API is aligned 🙌 
```suggestion
export const xmlLanguagePlugin: LanguagePlugin = {
```